planetarypraxis/smartforests
# Radio Player - Basic Features

This pull request adds radio player functionality.
- Radio player plays when play buttons are clicked either on an index or a page.
- Radio player player works as you might expect a radio player to work.

# Test Plan

1. Get this code on your machine.
2. Run `pipenv install` to download new dependencies.
3. Add a Radio index page.
4. Add a radio episode! You will need some MP3 files handy to do this.
5. Observe there is a index page and you can play the episode.
6. Observe you can play the episode on the individual episode page.

# Known limitations
- Needs tightening on mobile
